src/helpers/if-bool.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const compiledExpressions = {};
+
+function compile (expression) {
+	if (!/^((\$\d)| |(&&)|(\|\|)|\)|\()+$/.test(expression)) {
+		throw 'Unsafe expression passed to isBool helper';
+	}
+
+	if (expression.length > 20) {
```

Comment:
nope. but deliberately overly strict. Could loosen it up.

In actual fact I doubt there's a need for the limit as jsfuck requires square brackets, but just in case there's some other less verbose way of writing feature complete javascript with a subset of its characters I'm keeping a character limit.

Also probably a good thing to avoid these logic strings in templates getting too complex. the intention is mainly for relatively simple things like `if ((flag1 || flag2) && registeredUser)`